src/models.rs

```diff
@@ -0,0 +1,261 @@
+use std::collections::HashMap;
+
+use crate::{db_type::Result, table_definition::NativeModelOptions, ModelBuilder, ToInput};
+
+/// A collection of [`Model`](crate::Model) used by the [`Models`](crate::Models) to create
```

Comment:
Somewhere in this file I'd add a mention that usually there is little point to create models in run-time, and therefore it's good idea to define them with static lifetime, for example using lazy static crate.

Reply:
Done